src/debug.lisp

```diff
@@ -1,25 +1,25 @@
-(in-package #:coalton-impl)
+(in-package #:coalton)
 
-(defun print-value-db (&optional package)
```

Comment:
same here

this looks very bizarre; there must be a better way than to use this SBCL-specific extension